### ISSUE TITLE
Problem showing only 1 day in DayView

### ISSUE THREAD

**Issue by Awolize** [SimformSolutionsPvtLtd/flutter_calendar_view#93]
Before 1.0.0 ([534f2b7](https://github.com/SimformSolutionsPvtLtd/flutter_calendar_view/commit/534f2b7a541122572e66620d15707a3688498f85)) I could "hack" my way around this using: 
```Dart
DayView(  
minDay: DateTime.now(),  
maxDay: DateTime.now().add(const Duration(seconds: 1)),  
...
)
```

This is no longer possible. Is there a correct way of only displaying one day using the DayView?

**Comment by maxluetkemeyer** [comment]
Could you fix this issue shortly? Updated my Code to the new Version of the package, but just needed to revert it. :(
Anyways great package.

**Comment by faiyaz-shaikh** [comment]
Fixed in 1.0.1 PR #99
